CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.23.1]
+### Fixed
+- Fixed bug where Graph wouldn't render if figure.data wasn't set (#223)
```

Comment:
Similarly here - let's use the full link to the issue [#223](https://github.com/plotly/dash-core-components/issues/223)